src/mosse.py

```diff
@@ -39,7 +39,8 @@ def __init__(self, std=1.0, learning_rate=0.1):
 
     @staticmethod
     def normalize(patch):
-        patch = np.log(1+patch)
+        patch = patch + np.ones(patch.shape)
```

Comment:
Why was this necessary?

Reply:
I encountered some weird behavior of the normalizing function previously, in which only a subset of dimensions was used when adding the off-set. Consequentially, there was a slight chance of having entries with zeros in the resulting patch, rendering the log output undefined. I used this quick-fix to that problem, but I'm not sure if it still remains. Regardless, they should be equivalent.